sklearn/gaussian_process/kernels.py

```diff
@@ -737,6 +785,28 @@ class Product(KernelOperator):
     k2 : Kernel object
         The second base-kernel of the product-kernel
 
+    References
+    ----------
+    .. [1] `David Duvenau (2014). "The Kernel Cookbook:
+        Advice on Covariance functions".
+        <https://www.cs.toronto.edu/~duvenaud/cookbook/>`_
+
+    Examples
+    --------
+    >>> from sklearn.datasets import make_friedman2
+    >>> from sklearn.gaussian_process import GaussianProcessRegressor
+    >>> from sklearn.gaussian_process.kernels import RBF, Product, \
+                ConstantKernel
```

Comment:
You can instead do

``` python
    >>> from sklearn.gaussian_process.kernels import (RBF, Product,
                ConstantKernel)
```

Reply:
that gives a failed test though :(

Reply:
right, cause it needs a `...` at the beginning of the line.

``` python
    >>> from sklearn.gaussian_process.kernels import (RBF, Product,
    ...         ConstantKernel)
```